src/www/interfaces.php

```diff
@@ -2466,7 +2545,7 @@ function toggle_allcfg() {
                           </td>
                         </tr>
                         <tr class="dhcpv6_basic">
-                          <td><a id="help_for_dhcp6prefixonly" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Request only a IPv6 prefix"); ?></td>
+                          <td><a id="help_for_dhcp6prefixonly" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Request only an IPv6 prefix"); ?></td>
```

Comment:
probably best merged in a different PR

Reply:
I just saw it, the reason it is a separate commit, easy to extract and merge before ;)

Reply:
very forward-thinking... cherry-picked via 932038d36e